_layouts/statventures.html

```diff
@@ -41,7 +41,7 @@ <h1 class="visually-hidden">The future of data.</h1>
             <span>data.</span>
         </section>
         <p>StatVentures scouts technology through collaborations with innovators outside government to ensure the U.S. Census Bureau provides quality data more efficiently, creatively, and effectively.</p>
-        <a class="statventures-btn" href="http://statventuresinfosession.splashthat.com" target="_blank">Bring Us the Goods</a>
+        <a class="statventures-btn" href="https://www.challenge.gov/?challenge=statventures-supply-chain-p1" target="_blank">Bring Us the Goods</a>
```

Comment:
@hartkopfa Should the link need to be changed here is where to do so — in between the quotes of the href, exactly what is highlighted! Will add a comment to each link.